DynamicProgramming/EditDistance.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+Given two strings A and B, find the minimum number of steps required to convert A to B. (each operation is counted as 1 step.)
+
+You have the following 3 operations permitted on a word:
+
+Insert a character
+Delete a character
+Replace a character
+
+
+Input Format:
+
+The first argument of input contains a string, A.
+The second argument of input contains a string, B.
+Output Format:
+
+Return an integer, representing the minimum number of steps required.
+Constraints:
+
+1 <= length(A), length(B) <= 450
+
+
+Examples:
+
+Input 1:
+    A = "abad"
+    B = "abac"
+
+Output 1:
+    1
+
+Explanation 1:
+    Operation 1: Replace d with c.
+
+Input 2:
+    A = "Anshuman"
+    B = "Antihuman"
+
+Output 2:
+    2
+
+Explanation 2:
+    => Operation 1: Replace s with t.
+    => Operation 2: Insert i.
+    
+    
+https://www.interviewbit.com/problems/edit-distance/
+*/
+
+int Solution::minDistance(string A, string B) {
+    int m = A.size(), n = B.size();
+    int sol[m+1][n+1];
+    for(auto i=0;i<=m;++i){
```

Comment:
Could you please correct the spacing as was requested in StacksAndQueues/RainWaterTrapped.cpp ?

I'm not gonna comment for each line, please review and align the entire code in this file following the code styling of this repo. Thanks!